DigitalLearningSolutions.Web.Tests/ControllerHelpers/MockHttpContextSession.cs

```diff
@@ -0,0 +1,47 @@
+﻿namespace DigitalLearningSolutions.Web.Tests.ControllerHelpers
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Microsoft.AspNetCore.Http;
+
+    internal class MockHttpContextSession : ISession

```

Comment:
Would it not be sufficient/possible to mock the class using FakeItEasy (i.e. `A.Fake`)? 🙂

Reply:
I spent a while trying to get this to work. Unfortunately I depend on static extension methods in the `LearningMenuController` (in particular the `GetInt32` and `SetInt32` methods), which I don't think FakeItEasy supports 🙂 I might have been missing something though

Reply:
Ah I see! I did wonder if that was why. In that case creating our own mock sounds like a good solution 🙂